test/libsolidity/smtCheckerTests/special/msg_value_1.sol

```diff
@@ -0,0 +1,9 @@
+pragma experimental SMTChecker;
+contract C {
+	function f() public payable {
+		assert ((5 + msg.value + msg.value) - (4 + msg.value) > 0);
```

Comment:
Did you change the quadratic equation? :(

Reply:
I did, it was causing problems and this was more reliable, even though more boring =/